templates/summit/shared/_latest-blogs_summit.html

```diff
@@ -0,0 +1,46 @@
+<section class="p-section--deep">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <div class="p-section--shallow">
+      <h2>
+        <a href="{{ title_link }}">Latest from&nbsp;our&nbsp;blog&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+    <div class="p-section--shallow">
+      <div class="row" id="ubuntu-summit-latest"></div>
+    </div>
+  </div>
+
+  <template id="ubuntu-summit-template">
+    <div class="col-3 col-medium-2 u-equal-height-items">
+      <div class="u-crop--16-9">
+        <div class="article-image p-image-wrapper"></div>
```

Comment:
```suggestion
        <div class="article-image p-image-wrapper" aria-hidden></div>
```
As you are linking the images you will want to hide them from screen readers to avoid having the same link read twice

Reply:
That is really good suggestion.